modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -13,13 +13,17 @@ export interface Generators {
   readonly row: () => SugarElement;
   readonly replace: <K extends keyof HTMLElementTagNameMap>(cell: SugarElement, tag: K, attrs: Record<string, string | number | boolean | null>) => SugarElement;
   readonly gap: () => SugarElement;
+  readonly col: (prev: CellSpan) => SugarElement;
+  readonly colgroup: () => SugarElement;
 }
 
 export interface SimpleGenerators extends Generators {
   readonly cell: () => SugarElement;
   readonly row: () => SugarElement;
   readonly replace: (cell: SugarElement) => SugarElement;
   readonly gap: () => SugarElement;
+  readonly col: () => SugarElement;
+  readonly colgroup: () => SugarElement;
```

Comment:
It would be good, yes. From a quick look adding those types makes TableFill and BrowserTestGenerator not compile. Not sure what the best approach would be for either...

Reply:
I've fixed a number of them here, but there's still more to go which is outside the scope of this JIRA.